### DISCUSSION TITLE
DDR Approaching

### DISCUSSION THREAD

**Discussion by FrozenEmpire** [ChilliHugger/The-Lords-Of-Midnight#200] [Code Algorithm Questions] [answered]
Hi Chris,

Just to confirm, it is NOT possible to Approach a foe who is in the SAME location, only one that is in a location ahead, right ?

Regards,
Andrew

**Comment by IcemarkUK** [comment, marked as answer]
That's correct, approach does a walk forward, and then a failed approach enters battle.
